kerastuner/engine/hyperparameters.py

```diff
@@ -58,25 +58,31 @@ class Choice(HyperParameter):
         name: Str. Name of parameter. Must be unique.
         values: List of possible values.
             Any serializable type is allowed.
+        ordered: Whether the values passed should be considered to
+            have an ordering. This defaults to `True` for float/int
+            values and `False` for any other values.
         default: Default value to return for the parameter.
             If unspecified, the default value will be:
             - None if None is one of the choices in `values`
             - The first entry in `values` otherwise.
     """
 
-    def __init__(self, name, values, default=None):
+    def __init__(self, name, values, ordered=None, default=None):
         super(Choice, self).__init__(name=name, default=default)
         if not values:
             raise ValueError('`values` must be provided.')
         self.values = values
+        self.ordered = ordered
+        if self.ordered is None:
+            self.ordered = isinstance(values[0], (int, float))
```

Comment:
We should rather do this check for every value in `values`, or alternatively raise a TypeError if there is more than one type in `values`

Reply:
Done, adding a type check for int, float, str, or bool as well